install/kubernetes/helm/istio/charts/prometheus/values.yaml

```diff
@@ -56,3 +59,18 @@ service:
 
 security:
   enabled: true
+
+# Operator configuration is used to generate Custom Resources
+# for use with the https://github.com/coreos/prometheus-operator. This support
+# currently is only offered as an alpha feature.
+#
+# NOTE: Use of operator configuration is mutually-exclusive with the
+# templates to install the Istio prometheus addon.
+operator:
+  # Enabling this option will generate all of the ServiceMonitor CRs for Istio.
```

Comment:
Would it be better to have an `enabled` parameter here, rather than relying on checking the following two parameters to determine if the operator is being used? Would enable other parameters to be defined in the future (if necessary) without having to update the conditions in the deployment/configmap yamls - and could be a consistent approach for use of operators. Have used this approach in #13500.